problem/0001-automated-releases.md

```diff
@@ -0,0 +1,97 @@
+- Automated Releases
+- @kleimkuhler, @alpeb
+- 2020-02-17 (beginning of sprint)
+- 2020-03-03 (end of sprint)
+- RFC PR: [linkerd/rfc#7](https://github.com/linkerd/rfc/pull/7)
+- Linkerd Issue: [linkerd/linkerd2#3021](https://github.com/linkerd/linkerd2/issues/3021)
+- Reviewers: @grampelberg
+
+# Summary
+
+[summary]: #summary
+
+The goal for this project is to make the Linkerd2 release process as automated
+as possible. The process currently requires that the responsible member take
+multiple manual steps in order for the release artifacts to be properly built,
+tested, and updated for installations.
+
+Instead of addressing these points individually, the release process should be
+re-evaluated as a whole in order to utilize the CI workflow to automate the
+process where possible.
+
+# Problem Statement (Step 1)
+
+[problem-statement]: #problem-statement
+
+The manual steps required in order to perform a release can lead to any number
+of errors by fault of the responsible member. These include errors such as: 
+- forgetting to perform a particular step
+- completing the steps in an incorrect order
+- dealing with flakiness related to the timing of when steps are completed
+
+Additionally, the member must "babysit" the individual steps in order to start
+the next one. This can be time consuming.
+
+Once implemented, the release process will be:
+1. The responsible member opens a pull request that updates `CHANGES.md` with
+   the relevant changes for the upcoming release.
+2. Once step `1` is merged, the member pushes a signed tag for the release.
```

Comment:
I agree that signed tags are good, but we don't really express why they are necessary/what problem they solve. What's the goal of requiring signed tags? Also note that signed tags require a commit message (In Linkerd, this becomes the release notes). What is the expected format of this commit message?

Reply:
In 3be5245 I clarified the importance of a signed commit message for the tag and that it may be valuable in the future to verify the signee in `Design proposal`.